examples/DRISE.ipynb

```diff
@@ -503,20 +349,26 @@
    "source": [
     "sal_maps = app(\n",
     "    test_image_filename,\n",
-    "    blackbox_detect,\n",
-    "    rise_perturber,\n",
-    "    detect_alg,\n",
+    "    blackbox_detector,\n",
+    "    sal_generator,\n",
     "    fill=blackbox_fill,\n",
-    "    vis_mask_examples=False,\n",
     ")"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4e092529",
+   "metadata": {},
+   "outputs": [],
+   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "yolo",
```

Comment:
I'm not sure what we can do about these sections changing every time, if anything. If you think of anything (besides manually not committing these specific changes per-modified-notebook file, which is a little manual) I'm all ears.

Reply:
We should be able to create a filter that deletes the `kernelspec` entry of the metadata, which should make it default to Python3 kernel (accoring to this [issue](https://github.com/kynan/nbstripout/issues/141) that I found). I'm investigating this now.